src/index.js

```diff
@@ -84,3 +86,13 @@ Promise.all(
   localStorage.setItem("contributors", JSON.stringify(allContributors));
   generateCardGrid(allContributors);
 });
+
+const searchHacker = (e) => {
+  const filteredContributors = allContributors.filter((contrib) => {
+    const name = contrib.name || contrib.login;
+    return name.toLowerCase().includes(e.target.value.toLowerCase());
+  });
+  generateCardGrid(filteredContributors);
+};
+
+searchBox.addEventListener("input", searchHacker);
```

Comment:
Oh, python "sting" effect!

Reply:
```js
const name = contrib.name || contrib.login;
```
In the `or` operator, if the first part is true, it won't evaluate the second part. If `contrib.name` is `null` (which is falsy), it will evaluate `contrib.login` and store it in `name`.